pkg/processor/trigger/http/types.go

```diff
@@ -28,8 +29,11 @@ import (
 type Configuration struct {
 	trigger.Configuration
 	ReadBufferSize int
+	CORS           cors.CORS
```

Comment:
Perhaps consider CORS being a pointer? Where the default is CORS not populated... This might pollute encoded function configs.